test/mix/tasks/dogma_test.exs

```diff
@@ -31,23 +31,30 @@ defmodule Dogma.OptionParserTest do
       with "simple passed" do
         should "return simple formatter" do
           parsed = parse_args(["--format", "simple"])
-          assert parsed == {nil, Dogma.Formatter.Simple}
+          assert parsed == {nil, Dogma.Formatter.Simple, false}
         end
       end
 
       with "flycheck passed" do
         should "return flycheck formatter" do
           parsed = parse_args(["--format=flycheck"])
-          assert parsed == {nil, Dogma.Formatter.Flycheck}
+          assert parsed == {nil, Dogma.Formatter.Flycheck, false}
         end
       end
 
       with "unknown formatter passed" do
         should "return default formatter" do
           parsed = parse_args(["--format", "false"])
-          assert parsed == {nil, @default_formatter}
+          assert parsed == {nil, @default_formatter, false}
         end
       end
     end
+
+    with "noerrors option passed" do
+      should "return disabled error exit code" do
+        parsed = parse_args(["--noerrors"])
```

Comment:
Sorry, being picky again. Can we rename this to `--no-error` please, with a hyphen between the words. :)